src/app/page.tsx

```diff
@@ -1,6 +1,27 @@
+"use client";
+
+import { useState, useEffect } from "react";
 import DepartmentCard from "@/components/department-card";
+import ColaboradorCardDesktop from "@/components/socio-carddesktop";
+import ColaboradorCardMobile from "@/components/socio-cardmobile";
 
 export default function Home() {
+  const [windowWidth, setWindowWidth] = useState<number>(0);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowWidth(window.innerWidth);
+    };
+
+    handleResize();
+
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
```

Comment:
I challenge you to make this into a single component that changes its UI dynamically when the screen is mobile sized only with tailwind 👀 

I promise you the code will look so much cleaner and it will actually be easier in some way because we won't have to use this type of code, only css.